README.md

```diff
@@ -44,6 +44,19 @@ Or you can just run the following command:
 git config --global integrate.github-token <insert token here>
 ```
 
+git-integrate uses [git rerere](https://git-scm.com/docs/git-rerere) to resolve merge conflicts. To enable rerere, add the following to your `~/.gitconfig`:
+
+```
+[rerere]
+  enabled = true
+```
+
+Or run the following command:
+
+```bash
+git config --global rerere.enabled true
```

Comment:
Add documentation about a local repository config because some people may not want or understand rerere well enough to want it globally